TACC/Core-Styles
# feat(components): tup-491 eq. height cols via rows

## Overview

Add a class, `row-col-stretch-content-y`, to stretch bootstrap column content via row.

## Related

- supports https://github.com/TACC/tup-ui/pull/218
- builds upon #187

## Changes

- **added** `col-stretch-content-y` css and demo

## Testing

1. Open http://localhost:3000/components/detail/bootstrap--row.
2. Verify all columns' content (`.c-card`'s) are the same height.

## UI

<img width="1194" alt="row-col-stretch-content-y" src="https://github.com/TACC/Core-Styles/assets/62723358/49b0f658-eba7-4915-96c5-1121a543cbc1">
